README.md

```diff
@@ -1 +1,61 @@
-# Module1-Hello-Microverse
+# Project Name
+Hello microverseinc
+
+## Built With
+- Html 
+- Css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://ebukanweje.github.io/Module1-Hello-Microverse/)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
```

Comment:
- [ ] [OPTIONAL] I can see you are using the microverse template, good job 👏 but don't forget to customize all sections to your project.
Please fill this section. an example could be
```
## Setup

### Clone this repository


$ git clone git@github.com:EbukaNweje/Module1-Hello-Microverse.git
$ cd Module1-Hello-Microverse

### Open index.html with your browser
```